cpp/ps0005/problem solve 5.cpp

```diff
@@ -1,53 +1,56 @@
 #include <iostream>
 using namespace std;
-
-int main() 
-{ 
-    float num1 , num2;
+int main() {
+    // prompting user for 2 operands and an operator 2/2
+    float num1, num2;
     char operation;
     cout << "Enter first number:\n";
     cin >> num1;
-    cout << "Enter operation :\n";
+    cout << "Enter operation (+, -, *, /, % , ^):\n";
     cin >> operation;
     cout << "Enter second number:\n";
     cin >> num2;
 
+    // checking user inputs and processing 4/4
     double result;
-    switch (operation) {
-    case '+':
-    result = num1 + num2;
-    break;
-    case '-':
-    result = num1 - num2;
-    break;
-    case '*':
-    result = num1 * num2;
-    break;
-    case '^':
-    cout << "invalid character for operator \n";
-    return 1;
-    break;
-    case '%':
-    if ((num1 == (int)num1) && (num2 == (int)num2)) {
-    result = (int)num1 % (int)num2;
-    }else {
-    cout << "Cannot execute modulo operation on floating point numbers\n";
-    return 1;
-    }
-    break;
-    case '/':
-    if (num2 != 0) {
-    result = num1 / num2;
-    }
-    else {
-    cout << "Math eror: cannot divide by zero \n" ;
-    return 1;
-    }
-    break;
-    default:
-    cout << "Invalid !\n";
-    return 1;
-    }
-    cout << "Result:\n" << result ;
+    switch(operation) {
+        case '+':
+            result = num1 + num2;
+            break;
+        
+        case '-':
+            result = num1 - num2;
+            break;
+        
+        case '*':
+            result = num1 * num2;
+            break;
+        
+        case '/':
+            if (num2 != 0) {
+                result = num1 / num2;
+            } else {
+                cout << "Math eror cannot divide by zero \n" ;
+                return 1;
+            } // 1/1
+            break;
+        
+        case '%':
+            if ((num1 == (int)num1) && (num2 == (int)num2)) {
+            result = (int)num1 % (int)num2;
+            } else {
+                cout << "Cannot execute modulo operation on floating numbers\n";
+                return 1;
+            } // 1/1
+            break;
+        
+        default:
+            cout << "Invalid operation!\n";
+            return 1; // 1/1
+    } // 1/1

```

Comment:
- [x] used `switch` and `if` structures in a right way to handle different cases
- [x] handled division operation `/`
- [x] handled modulo operation `%`
- [x] handled invalid operator case
- [x] chose the right operation for each case